### ISSUE TITLE
Shouldn't the normalization layer come before the Relu activation?

### ISSUE THREAD

**Issue by issa-s-ayoub** [philipperemy/keras-tcn#12]
https://github.com/philipperemy/keras-tcn/blob/c6799257842cf65ee5fd46a921d4c50c9d1b9b87/tcn/tcn.py#L30

I can find that in the original paper, the normalization layer comes before the Relu activation layer. Why this isn't the case here?

Thank you!!

**Comment by philipperemy** [comment]
First, I replaced weight normalization with layer normalization. They won’t be the same, but should have similar effects in stabilizing training.

You cannot place weight normalization after ReLU because it normalizes the cell weights, as the name suggested, not the cell output.

As the author of the paper stated, this TCN architecture really aims to be a simple baseline that you can improve upon. So personally I think replicating the exact same results is not that important.
